src/query/storages/common/table-meta/src/caches/meta_cache_manager.rs

```diff
@@ -58,8 +57,11 @@ impl CacheManager {
             let table_snapshot_cache = Self::new_item_cache(config.table_cache_snapshot_count);
             let table_statistic_cache = Self::new_item_cache(config.table_cache_statistic_count);
             let segment_info_cache = Self::new_item_cache(config.table_cache_segment_count);
+            // TODO remove this
+            // let bloom_index_data_cache =
+            //    Self::new_bytes_cache(config.table_cache_bloom_index_data_bytes);
             let bloom_index_data_cache =
-                Self::new_bytes_cache(config.table_cache_bloom_index_data_bytes);
+                Self::new_item_cache(config.table_cache_bloom_index_data_bytes);
```

Comment:
Is the capacity `table_cache_bloom_index_data_bytes` not correct?

Reply:
thanks for reminding me